### PR TITLE
Add static content to Individual PSC Statement

### PR DESCRIPTION
**Pull request by eedwards0**
IDVA3-1037: Add static content to 'Individual PSC statement' screen
- Static content added with localisation
- Added placeholder for dynamic text [name] within the statement
- Renamed the 'skeleton_five' files and references to 'individual statement'
- Updated link on the personal code page to reference the individual PSC statement page

### CODE REVIEW

File: src/views/router_views/individual_statement/individual_statement.njk [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          text: i18n.individual_statement_part1 + "[name]" + i18n.individual_statement_part2
+        }
+      ]
+    }) }}  
+
+    {{ govukButton({
+        text: i18n.global_save_continue_button,
+        href: Urls.PSC_VERIFIED,
+        classes: "govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8"
+    }) }}
```
Comment (PhilipGlover) [inline src/views/router_views/individual_statement/individual_statement.njk:29]: I'm just doing a fix for id on other screens now.  Can you add an id to this button for tester's automation?

File: src/views/router_views/individual_statement/individual_statement.njk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          value: i18n.individual_statement_value,
+          text: i18n.individual_statement_part1 + "[name]" + i18n.individual_statement_part2
+        }
+      ]
+    }) }}  
+
+    {{ govukButton({
+        text: i18n.global_save_continue_button,
+        href: Urls.PSC_VERIFIED,
+        classes: "govuk-button--start govuk-!-margin-top-2 govuk-!-margin-bottom-8"
```
Comment (PhilipGlover) [inline src/views/router_views/individual_statement/individual_statement.njk:28]: I don't think we need this class `govuk-button--start`

File: src/views/router_views/individual_statement/individual_statement.njk [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
```
Comment (PhilipGlover) [inline src/views/router_views/individual_statement/individual_statement.njk:5]: I think we need `{% set title = i18n.individual_statement_title %}` in order for the browser tab to display the title.
(`title` gets used in `partials/__meta_header.njk`

File: src/views/router_views/individual_statement/individual_statement.njk [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{% block main_content %}
+
```
Comment (PhilipGlover) [inline src/views/router_views/individual_statement/individual_statement.njk:7]: Do we want to add `[name]` above the title as per the prototype?  I know it'll have to be done properly, but just as a placeholder?

**Review by PhilipGlover — CHANGES_REQUESTED** [review]
- needs a merge from main
- few comments re the template

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 2 above)
Comment (eedwards0) [reply src/views/router_views/individual_statement/individual_statement.njk:28]: I started this way, but would this be covered by the 'isPageHeading: true' attribute in the govukCheckboxes?

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 4 above)
Comment (eedwards0) [reply src/views/router_views/individual_statement/individual_statement.njk:7]: Yes :)

Comment (PhilipGlover) [reply src/views/router_views/individual_statement/individual_statement.njk:7]: I'm struggling to see a GDS component that would place the name above the title.  The checkbox component places a hint below the heading.
Either we mention this to UX or I guess we can just use plain html with a `govuk-caption-xl` class ???

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 4 above)
Comment (eedwards0) [reply src/views/router_views/individual_statement/individual_statement.njk:7]: Got this working locally.

File: src/views/router_views/individual_statement/individual_statement.njk [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        {
+          value: i18n.individual_statement_value,
+          text: i18n.individual_statement_part1 + "[name]" + i18n.individual_statement_part2
+        }
+      ]
+    }) }}  
+
+    {{ govukButton({
+        id: "confirmContinue",
+        text: i18n.global_save_continue_button,
```
Comment (PhilipGlover) [inline src/views/router_views/individual_statement/individual_statement.njk:30]: sorry, missed before.  This should be `global_confirm_continue_button`

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 4 above)
Comment (PhilipGlover) [reply src/views/router_views/individual_statement/individual_statement.njk:8]: It's below the heading though.. see my comment above

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 3 above)
Comment (PhilipGlover) [reply src/views/router_views/individual_statement/individual_statement.njk:6]: I don't see this added yet

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 3 above)
Comment (eedwards0) [reply src/views/router_views/individual_statement/individual_statement.njk:6]: Check now :)

Comment (PhilipGlover) [reply src/views/router_views/individual_statement/individual_statement.njk:6]: 👍

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/router_views/individual_statement/individual_statement.njk (hunk 4 above)
Comment (PhilipGlover) [reply src/views/router_views/individual_statement/individual_statement.njk:8]: the ts file formatting is slightly off, and the actual elements and classes may change later, but meets what we need now.

**Review by PhilipGlover — APPROVED** [review]
- matches prototype as far as we can at this stage
- code looks as expected

Screenshot 2024-03-08 at 1 52 31 PM

**Pull request merged**